contrib/node/src/python/pants/contrib/node/tasks/node_repl.py

```diff
@@ -70,3 +70,9 @@ def launch_repl(self, targets):
 
         repl_session = node_repl.run()
         repl_session.wait()
+    # TODO(qsong): Consider not specifying dependency in BUILD
+    # Repl task doesn't make any sence for Node.js with yarnpkg for the following reasons:
+    # 1. Node.js can simply start from the source root, make an alias for 'cd package_root && node'
+    #   does not give any benefits but increase the maintance work
+    # 2. There's no simple entry point (binary) for Node.js packages. A package may start with node,
```

Comment:
Node.js have different repl binaries. Invoking "node" in cli directly is one way which I don't think is quite useful for node developers. More usable repl may be "babel-node" which will live in ./node_modules/.bin/ after installing the dependencies. There are other binaries for the packages. The most important thing is that since node program is self-contained, developers can always start a repl under their code directory. It would be very hard to sell this idea to node developers by restricting their extensibility to develop fast.